streams/us.m3u

```diff
@@ -393,6 +393,8 @@ https://ov.ottera.tv/live/master.m3u8?channel=elec_en
 https://hls-cdn.tvstartup17.net/tvstartup17-channel/live/mp4:emtmediaedge/playlist.m3u8
 #EXTINF:-1 tvg-id="EntrepreneurTV.us",Entrepreneur (720p)
 https://a.jsrdn.com/broadcast/7582ed85f7/+0000/c.m3u8
+#EXTINF:-1 tvg-id="EntertainmentTonight.us",Entertainment Tonight (720p) [Geo-blocked]
+https://etlive-mediapackage-fastly.cbsaavideo.com/dvr/manifest.m3u8
```

Comment:
got 403 even from Miami

Reply:
https://www.cbsnews.com/colorado/xml-sitemap-video/cbs-news-live/2023-11.xml

https://www.cbsnews.com/colorado/xml-sitemap-video/cbs-news-live/2024-01.xml

Seems to have indicated this link.

Reply:
@Alstruit @PopeyeTheSai10r

Reply:
I'm getting a 403 on this: https://etlive-mediapackage-fastly.cbsaavideo.com/dvr/manifest.m3u8
I think they only allow the dai.google.com to start because if you see the .ts files are being served from:
https://cbsta49f-cbsta49f-ms.global.ssl.fastly.net/amagi7b98-AmagiMixible/master/amagi7b98-AmagiMixible_6/00797/amagi7b98-AmagiMixible_6_20240213T172144_6_00022.ts

https://www.cbsnews.com/philadelphia/xml-sitemap-video/cbs-news-live/2024-01.xml

Reply:
Got 403 from ATL.

Reply:
https://dai.google.com/linear/hls/event/xrVrJYTmTfitfXBQfeZByQ/master.m3u8 works